docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -165,77 +147,6 @@ list_adapters_component_wise
 {"text_encoder": ["toy", "pixel"], "unet": ["toy", "pixel"], "text_encoder_2": ["toy", "pixel"]}
 ```
 
-## Compatibility with `torch.compile`
```

Comment:
Why is this going away? 😱

Reply:
Oops sorry I forgot about this section! Added it back in as a nested section under the `fuse_lora` section 😅

Reply:
I am a little uncomfortable about how it's ported in the new section. 

Left a comment here: https://github.com/huggingface/diffusers/pull/7213/files#r1513791539.